wp-content/themes/core/blocks/tribe/terms/style.pcss

```diff
@@ -0,0 +1,82 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+/* -----------------------------------------------------------------------
+ * Terms Block
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+
+	&.aligncenter {
+		justify-content: center;
+	}
+
+	&.alignright {
+		justify-content: flex-end;
+	}
+}
+
+/* -----------------------------------------------------------------------
+ * Terms Block - List
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms__list {
+	display: inline-flex;
+	align-items: center;
+	gap: 8px;
+	padding: 0;
+	margin: 0;
+	list-style-type: none;
+}
+
+/* -----------------------------------------------------------------------
+ * Terms Block - Link (sometimes not a link)
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms__link {
+
+	@mixin t-category;
+	color: var(--color-black);
+
+	/* target links for the hover/focus effect */
+	&:is(a) {
+
+		&:hover,
+		&:focus {
+			color: var(--color-black);
+			text-decoration: underline;
+		}
+	}
+
+	/* style pills differently */
+	.is-style-pills & {
+
+		@mixin t-tag;
+		display: inline-flex;
+		padding: 4px 16px;
+		align-items: center;
+		background-color: var(--color-neutral-20);
+		border-radius: 100px;
+		color: var(--color-black);
+		text-decoration: none;
+		transition: var(--transition);
+
+		/* target links for the hover/focus effect */
+		&:is(a) {
+
+			&:hover,
+			&:focus {
+				background-color: var(--color-black);
+				color: var(--color-white);
+				text-decoration: none;
+			}
+		}
+	}
+}
```

Comment:
Since we can control the markup & classes for this block, I'd love to see us use utility classes for these styles rather than a set of one-off styles specific to this block.

Reply:
Do you have any ideas on where these styles would live in the global settings? Maybe mixins/utilities in `pcss/cards`? Are you just looking for utilities specific to the pill styling?

Reply:
I'd say adding them to the typography mixins & helpers like your existing t-tag mixin. The key is creating both the mixin and a related helper/utility class. That way any time we have the ability to add classes ourselves we can just use the utility class, instead of defaulting to the mixin. (Since mixins duplicate the styles, which bloats the overall amount of css being rendered.)

Reply:
I've updated this so that the term links default to have the `t-category` class, removing that extra CSS. If the terms are set to display as pills, the styles should override with that mixin. Also added utility classes that were missing from some mixins.